### PR TITLE
Lvrubym 334: Configure Karma, Jasmine

### PR DESCRIPTION
**Pull request by yaremakleputs**
## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVRUBYM-334)
* [JIRA subtask](http://ssu-jira.softserveinc.com/browse/LVRUBYM-334)

## Code reviewers

- [x] @andriyarterchuk 
- [x] @VitaliaDurnyak 
- [ ] @Mykhailo123 

### Second Level Review

- [ ] @4asof4ik

## Summary of issue

Need to configure Karma

## Summary of change

- Configure Karma and add another gems

## Testing approach

Test with JSCS

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Karma configuration
+// Generated on Thu Apr 20 2017 22:09:07 GMT+0300 (EEST)
```
Comment (Vovk-Nazar) [inline karma.conf.js]: You realy need this information

File: test/spec/test.spec.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe("A suite", function() {
```
Comment (Vovk-Nazar) [inline test/spec/test.spec.js]: Em 0_o

File: package.json [hunk 3]
```diff
@@ -6,12 +6,22 @@
   "scripts": {
```
Comment (skrypnyk88) [inline package.json]: Will we use angular-mocks?

File: karma.conf.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      './specs.webpack.js': ['webpack', 'coverage']
+    },
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: true,
+    browsers: ['PhantomJS'],
+    singleRun: true,
+    concurrency: Infinity
+  })
```
Comment (skrypnyk88) [inline karma.conf.js]: lost semicolon

File: karma.conf.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+}
```
Comment (skrypnyk88) [inline karma.conf.js]: lost semicolon

File: karma.conf.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    reporters: ['progress', 'coverage'],
+    preprocessors: {
+      './specs.webpack.js': ['webpack', 'coverage']
+    },
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: true,
+    browsers: ['PhantomJS'],
+    singleRun: true,
```
Comment (4asof4ik) [inline karma.conf.js]: this option is in package.json as --single-run

Comment (yaremakleputs) [inline karma.conf.js]: You are right. I will delete it, because "false" is default value.

**Pull request merged**